partner_address_two_lines/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_contact_name(self, partner, name):
+        if self.env.context.get("_two_lines_partner_address"):
```

Comment:
If you return the result from super when _two_lines_partner_address is NOT in context, you do not need the else and save an indentation level for the remainder of the method.